code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -19,36 +8,91 @@
 	description = "A well-known alcohol with a variety of applications."
 	color = "#404030" // rgb: 64, 64, 48
 	nutriment_factor = 0
-	var/boozepwr = 10 //lower numbers mean the booze will have an effect faster.
+	var/boozepwr = 65 //Higher numbers equal higher hardness, higher hardness equals more intense alcohol poisoning
+
+/*
+Boozepwr Chart
+Note that all higher effects of alcohol poisoning will inherit effects for smaller amounts (i.e. light poisoning inherts from slight poisoning)
+In addition, severe effects won't always trigger unless the drink is poisonously strong
+All effects don't start immediately, but rather get worse over time; the rate is affected by the imbiber's alcohol tolerance
+
+0: Non-alcoholic
+1-10: Barely classifiable as alcohol - occassional slurring
+11-20: Slight alcohol content - slurring
+21-30: Below average - imbiber begins to look slightly drunk
+31-40: Just below average - no unique effects
+41-50: Average - mild disorientation, imbiber begins to look drunk
+51-60: Just above average - disorientation, vomiting, imbiber begins to look heavily drunk
+61-70: Above average - small chance of blurry vision, imbiber begins to look smashed
+71-80: High alcohol content - blurry vision, imbiber completely shitfaced
+81-90: Extremely high alcohol content - heavy toxin damage, passing out
+91-100: Dangerously toxic - extreme toxin damage
+*/
 
 /datum/reagent/consumable/ethanol/on_mob_life(mob/living/M)
-	M.jitteriness = max(M.jitteriness-5,0)
-	if(current_cycle >= boozepwr*0.5)
-		var/drunk_value = sqrt(volume*1000/boozepwr)
-		if(volume >= boozepwr*0.2)
-			if(M.slurring < drunk_value)
-				M.slurring += 4
-			M.Dizzy(drunk_value)
-		if(volume >= boozepwr*0.8)
-			if(M.confused < drunk_value)
-				M.confused += 3
-		if(volume >= boozepwr*3.8)
-			M.adjustToxLoss(1, 0)
-			. = 1
+	if(M.alcohol_tolerance == COMPLETE_TOLERANCE)
+		return ..() || .
+	var/alcohol_effect = (boozepwr * (current_cycle / 50)) / M.alcohol_tolerance
```

Comment:
Tried to use a switch() here but it didn't want to work correctly so this is probably awful.